autogpt_platform/frontend/src/components/nav/NavBarButtons.tsx

```diff
@@ -0,0 +1,74 @@
+"use client";
+
+import React from "react";
+import Link from "next/link";
+import { BsBoxes } from "react-icons/bs";
+import { LuLaptop, LuShoppingCart } from "react-icons/lu";
+import { cn } from "@/lib/utils";
+import { usePathname } from "next/navigation";
+import { getBehaveAs } from "@/lib/utils";
+import MarketPopup from "./MarketPopup";
+
+export function NavBarButtons({ className }: { className?: string }) {
+  const pathname = usePathname();
+  const buttons = [
+    {
+      href: "/",
+      text: "Monitor",
+      icon: <LuLaptop />,
+    },
+    {
+      href: "/build",
+      text: "Build",
+      icon: <BsBoxes />,
+    },
+  ];
+
+  const isCloud = getBehaveAs() === "CLOUD";
```

Comment:
Since we ended up going with hard-coding the route,
Create an enum for CLOUD ?

Reply:
Done